.github/workflows/deploy-to-s3.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy To S3
+
+on:
+  push:
+    branches: [ official ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        aws-access-key-id: ${{ secrets.CI_CD_AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.CI_CD_AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+    - name: Install dependencies
+      run: |
```

Comment:
`wca-documents` has the approach of using an `install-dependencies.sh` script, whereas here you're installing stuff inline in the GitHub action. Can you decide on which is better and enforce it consistenly?

(I'm leaning towards installing stuff inline in the YML, in which case I should have probably made this comment over on the other PR...)

Reply:
yeah I'll make everything inline